fw/http_limits.c

```diff
@@ -241,6 +244,8 @@ frang_conn_limit(FrangAcc *ra, FrangGlobCfg *conf)
 			frang_limmsg("new connections rate",
 				     csum, conf->conn_rate,
 				     &FRANG_ACC2CLI(ra)->addr);
+			BUG_ON(!ra->conn_curr);
```

Comment:
Here and at the below I think it's better to use `WARN_ON()` or even `WARN_ON_ONCE()`: it seems we do not crash if `!ra->conn_curr` and we'll notice the problem in dmesg any way, but this `BUG_ON()` may lead to a crash.

Reply:
Done